plugin.video.botallen.hotstar/resources/lib/main.py

```diff
@@ -69,6 +69,10 @@ def login(plugin):
         pdialog.update(i, line3="[B][UPPERCASE]%s[/UPPERCASE][/B]" % code)
     pdialog.close()
 
+@Script.register
```

Comment:
```suggestion
@Script.register
```

Also this is not required